src/ddmd/backend/code.d

```diff
@@ -217,6 +217,7 @@ extern __gshared int refparam;
 targ_size_t cod3_spoff();
 uint calccodsize(code *c);
 targ_size_t cod3_bpoffset(Symbol *s);
+void searchfixlist(Symbol *s) { }
```

Comment:
This too.

Reply:
#7150 explains why that is there.

Reply:
Thanks, I missed Martin's comment there.